### PR TITLE
feat(checkout): CHECKOUT-7756 Update docs around item-level discount

### PR DESCRIPTION
**Pull request by valentindellangela**
# [DEVDOCS-]
[{Ticket number or summary of work}](https://bigcommercecloud.atlassian.net/browse/CHECKOUT-7756)

## What changed?
Provide a bulleted list in the present tense
* V3/Discounts API can now be used to add item-level discount in addition to cart level discount
* You cannot apply a manual discount against a specific line item if a coupon/automatic promotion is already applied to this item

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.
- We can't publish this PR until the project has been completed as it involves multiple teams working together - That's why I created a Draft PR

## Testing?
image

ping @bigcommerce/team-checkout  @bigcommerce/dev-docs-collaborators

### CODE REVIEW

**Comment by valentindellangela** [comment]
I updated this PR to include this new limitation
> You cannot apply a manual discount against a specific line item if a coupon/automatic promotion is already applied to this item

File: reference/checkouts.v3.yml [hunk 1]
```diff
@@ -1599,12 +1599,18 @@ paths:
...
-        This endpoint splits the discount between line items based on the item value.
+        This endpoint can be used to apply:
+        * A manual discount to a cart. In this case, the discount will be distributed between each line items in the cart based on the item value.
+        * And/Or to apply a manual discount against a specific line item.
+        
+        Manual discount at the item level can be added along with a cart level discount.
+        
+        Notes:
+        * When you call this API, you clear out all existing discounts applied to line items, including product and order-based discounts.
+        * You cannot apply a manual discount against a specific line item if a coupon/automatic promotion is already applied to this item
```
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:1610]:
```suggestion
        * You cannot apply a manual discount against a specific line item if you have already applied a coupon/automatic promotion to this item.
```

**Review by bc-traciporter — COMMENTED** [review]
Made some editorial suggestions

**Comment by slsriehl** [comment]
@valentindellangela Please reopen when ready using successor branch https://github.com/bigcommerce/docs/tree/CHECKOUT-7756.

**Pull request closed without merging**